src/Lambda/LambdaClient.php

```diff
@@ -67,4 +69,40 @@
  * @method \Aws\Result updateFunctionConfiguration(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateFunctionConfigurationAsync(array $args = [])
  */
-class LambdaClient extends AwsClient {}
+class LambdaClient extends AwsClient
+{
+    /**
+     * Default Curl options for Lambda
+     * @var array
+     */
+    static $defaultCurlOptions = [
+        CURLOPT_TCP_KEEPALIVE => 1,
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(array $args)
+    {
+        parent::__construct($args);
+        $list = $this->getHandlerList();
+        $list->appendInit($this->getDefaultCurlOptionsMiddleware());
+    }
+
+    /**
+     * Provides a middleware that sets default Curl options for the command
+     *
+     * @return callable
+     */
+    public function getDefaultCurlOptionsMiddleware()
+    {
+        return Middleware::mapCommand(function (CommandInterface $cmd) {
+            if (!isset($cmd['@http']['curl'])) {
```

Comment:
Is this wanted for all operations or just a subset?

Reply:
Current scope is for all operations.